test/integration/examples/apiserver_test.go

```diff
@@ -292,6 +292,8 @@ func TestAggregatedAPIServer(t *testing.T) {
 			"--etcd-servers", framework.GetEtcdURL(),
 			"--cert-dir", wardleCertDir,
 			"--kubeconfig", wardleToKASKubeConfigFile,
+			"--emulated-version", "wardle=1.1",
+			"--feature-gates", "wardle:BanFlunder=true",
```

Comment:
good test. Can you also check the `--emulated-version=wardle:1.1` without the `--feature-gates` and see if the `"badname"` can be created?

Reply:
added another test without the `--feature-gates` flag